logisland-components/logisland-services/logisland-service-elasticsearch/logisland-service-elasticsearch_7_x-client/src/main/java/com/hurence/logisland/service/elasticsearch/Elasticsearch_7_x_ClientService.java

```diff
@@ -615,16 +620,42 @@ public List<MultiGetResponseRecord> multiGet(List<MultiGetQueryRecord> multiGetQ
                     Map<String,Object> responseMap = response.getSourceAsMap();
                     Map<String,String> retrievedFields = new HashMap<>();
                     responseMap.forEach((k,v) -> {if (v!=null) retrievedFields.put(k, v.toString());});
+                    getLogger().trace("multiGet response responseMap="+responseMap+" retrievedFields="+retrievedFields);
                     multiGetResponseRecords.add(new MultiGetResponseRecord(response.getIndex(), response.getType(), response.getId(), retrievedFields));
                 }
             }
         }
 
+        if (getLogger().isTraceEnabled()) {
+            getLogger().trace("multiGet response "+multiGetResponseRecords);
+        }
+
         return multiGetResponseRecords;
     }
 
     @Override
     public MultiQueryResponseRecord multiQueryGet(MultiQueryRecord queryRecords) throws DatastoreClientServiceException{
+        final List<QueryRecord> queries = queryRecords.getQueries();
```

Comment:
Once again you need to test that

Reply:
FOr exemple a risk is that one of the request is not made because of a bug in the algorithm (which is not easy to see for a human).

Reply:
Added test which check that request is performed in chunks.